README.md

```diff
@@ -44,7 +44,7 @@ The command-line program, `dtmate` *(along with the golang package)* allows you
 ## Installation
 
 * Library: `go get -u github.com/jftuga/DateTimeMate`
-* Command line tool: `go install github.com/jftuga/DateTimeMate/cmd/dtmate@latest`
+* Command line tool: `go install -ldflags="-s -w" github.com/jftuga/DateTimeMate/cmd/dtmate@latest`
```

Comment:
I'm perplex about the flags here, is it because of the embed.FS ?

Reply:
No.  It just makes the binary executable size smaller when you run the `go install` command.